src/main.cpp

```diff
@@ -38,16 +110,23 @@ int main() {
         bot.getApi().sendMessage(message->chat->id, "Unknown command: " + message->text);
     });
 
-    bot.getEvents().onCommand("start", [&bot](TgBot::Message::Ptr message) {
+    bot.getEvents().onCommand("start", [&bot, &users](TgBot::Message::Ptr message) {
+        if (HandleUser(message->chat->id, users)) {
```

Comment:
Confusing as I expected that this function is void by its name.

Reply:
renamed to CheckDDOSUser